CMakeLists.txt

```diff
@@ -93,13 +93,13 @@ elseif( CMAKE_CXX_COMPILER MATCHES ".*/hipcc$" )
   set( CMAKE_C_COMPILE_OPTIONS_VISIBILITY_INLINES_HIDDEN "-Xcompiler ${CMAKE_C_COMPILE_OPTIONS_VISIBILITY_INLINES_HIDDEN}" )
   set( CMAKE_CXX_COMPILE_OPTIONS_VISIBILITY_INLINES_HIDDEN "-Xcompiler ${CMAKE_CXX_COMPILE_OPTIONS_VISIBILITY_INLINES_HIDDEN}" )
 elseif( CMAKE_CXX_COMPILER MATCHES ".*/hcc$" )
-  message( STATUS "HCC compiler set; ROCm backend selected [ CXX=/opt/rocm/bin/hcc cmake ... ]" )
+  message( STATUS "HCC compiler set; ROCm backend selected [ CXX=/opt/rocm-<ver>/bin/hcc cmake ... ]" )
```

Comment:
This line looks broken, how will it work if the user wants to use /opt/rocm/ only?

Reply:
As I updated in the other comment, the symlink is needed only when user is using frameworks like pytorch/Caffe2 or any benchmark framework. It helps to keep an uniform interface when single/multiple ROCm versions are installed on a machine.
In the above scenario while compiling the rocFFT it is required to point to the exact path of ROCm so that we pick the correct version of compiler/hcc binary.